src/sagas/backend.ts

```diff
@@ -406,8 +417,11 @@ async function postAssessment(id: number, tokens: Tokens): Promise<Response | nu
  * GET /grading
  * @returns {Array} GradingOverview[]
  */
-async function getGradingOverviews(tokens: Tokens): Promise<GradingOverview[] | null> {
-  const response = await request('grading', 'GET', {
+async function getGradingOverviews(
+  tokens: Tokens,
+  group = false
```

Comment:
Could this be a parameter with no default value? It's safer behaviour, and helps with figuring out issues.

Reply:
Just noticed that you use this function at both `fetchGrading` and `fetchGradingOverview`, that's nice

Reply:
Haha refactored as well 

(Although I was forced to use default values for my action to use `ContentDisplay`)